### PR TITLE
Feature/create project with owner

### PR DESCRIPTION
**Pull request by harlanj**
- Update `project create` command to include project-owner prompt
- Add librarian call to get user orgs
- Update project.create to check for `--project-owner` flag

### CODE REVIEW

**Comment by harlanj** [comment]
Currently waiting on Codeship to push out some code that relates to user CI with Modulus

**Comment by fiveisprime** [comment]
:shipit:

File: lib/commands/project.js [hunk 1]
```diff
@@ -352,6 +344,23 @@ project.create = function (projectName, servoSize, runtime, cb) {
         fn();
       });
     },
+    function findOrganizationMembers(fn) {
+      userController.getOrganizations(userConfig.data.userId, function(err, results) {
```
Comment (franvarney) [inline lib/commands/project.js]: Space after `function`

File: lib/commands/project.js [hunk 2]
```diff
@@ -1163,11 +1278,10 @@ project.chooseProjectPrompt = function (projects, projectName, cb) {
     if (meta && meta['mod-project-name'] && !projectName) {
       // "mod-project-name" found in package, check for the project name
       // in the list of projects to make sure it exists.
-      projectIndex = projectInArray(projects, meta['mod-project-name']);
-
-      if (projectIndex >= 0) {
-        modulus.io.print('Found mod-project-name \'' + meta['mod-project-name'] + '\' in package.json');
-        projectName = projects[projectIndex].name;
+      exists = projects.some(function(p) {return p.name.toUpperCase() === meta['mod-project-name'].toUpperCase();});
```
Comment (franvarney) [inline lib/commands/project.js]: Use multiple lines

File: lib/commands/project.js [hunk 3]
```diff
@@ -569,8 +633,9 @@ project.deploy = function (projectName, dir, includeModules, registry, meteorDeb
       });
     },
     function promptForProject (projects, fn) {
-      if (projectName && projectInArray(projects, projectName) === -1) {
-        return fn(null, projects, null);
+      if (projectName) {
+        var exists = projects.some(function(p) {return p.name.toUpperCase() === projectName.toUpperCase();});
```
Comment (franvarney) [inline lib/commands/project.js]: Use multiple lines

**Comment by franvarney** [comment]
Lots of areas that could have space after a function

File: lib/commands/project.js (hunk 1 above)
Comment (harlanj) [inline lib/commands/project.js]: `mod-standard` is reporting `Unexpected space before function parentheses` when I add the space there.

Comment (fiveisprime) [inline lib/commands/project.js]: No space after named functions (it's already there).

```
function name() {
        /\
       space
        \/
function () {
```

:ok_hand:

**Comment by harlanj** [comment]
Some of the changes you want made are outside of the scope of this branch. We should open another issue to lint the project. I merged master into this branch to get up-to-date.

**Comment by musgravejw** [comment]
:+1:

**Comment by tzmanics** [comment]
@harlanj if you resolve the conflicts I'll test & review :)

**Comment by jackboberg** [comment]
rebased locally from master, testing..

**Comment by jackboberg** [comment]
This prompts `Please choose the owner for this project:` even if the user is not in any orgs. I think this should default the current user unless there are options.

@fiveisprime ?

**Comment by jackboberg** [comment]
👍 `modulus project open`

**Comment by fiveisprime** [comment]
Please no prompt when no orgs.

**Comment by jackboberg** [comment]
👍 no prompt, will tag as `6.2.0-beta` and publish as `next`

**Pull request merged**